From d47905b513a4e7f2b369d90c2b54b2d755e942a9 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 13:59:24 -0700
Subject: [PATCH 539/830] _____ANNOTATION: Android kconfigs and build
 scripts_____
X-NVConfidentiality: public

Everything else, mainly Android-specific kernel configuration and build
scripts, e.g. gki_defconfig, build.*, Kconfig.gki.  Avoid putting
changes to upstream files in this category.

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index c62e1d7fa95d..c01635f7d69e 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -29,3 +29,4 @@ Incremental filesystem
 networking changes
 hardware-wrapped key support
 db845c (DragonBoard 845c) support
+Android kconfigs and build scripts
-- 
2.17.1

